dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/alert/ProcessAlertManager.java

```diff
@@ -410,6 +441,14 @@ public void sendProcessBlockingAlert(ProcessInstance processInstance,
         Alert alert = new Alert();
         String cmdName = getCommandCnName(processInstance.getCommandType());
         List<ProcessAlertContent> blockingNodeList = new ArrayList<>(1);
+
+        ProcessDefinitionLog processDefinitionLog = processDefinitionLogMapper
+                .queryByDefinitionCodeAndVersion(processInstance.getProcessDefinitionCode(),
+                        processInstance.getProcessDefinitionVersion());
+        Map<Integer, String> userMap = userMapper.selectList(new QueryWrapper<>()).stream()
```

Comment:
May I ask Why use a map to store user information instead of querying directly with user ID?

Reply:
I referenced the code from version 3.1.7 to get the operator information, like https://github.com/apache/dolphinscheduler/blob/3.1.7-release/dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java#L503 ,let me optimize it a bit

Reply:
> May I ask Why use a map to store user information instead of querying directly with user ID?

I have optimize, thanks for the reminder.